### PR TITLE
Advanced Security Cameras show cyborg cameras, just like their non-advanced counterpart.

### PR DESCRIPTION
**Pull request by ghost**
Despite Adv Cameras now working some people still used their non-advanced counterpart because it shows cyborgs while the advanced ones don't.
This was surprisingly easy to do, since the HTMLUI was made for the crew monitoring console which does this thing to begin with.
Tested that they update the absolute least necessary (using optimizations from the crew monitoring console, i.e. only update when someone's watching the console, queues updates from all cyborgs together every 3 seconds) and that unlinked cyborgs don't show up in the map.

Also, I couldn't test 2+ cyborgs moving around at the same time. If only one of them updates, it's an easy fix, but I can't test it myself.

To do some other time: It might get a little hard to spot the cyborg icons when you specifically want to watch cyborgs only, so I could either make the cyborgs another color, or add the text-based menu that never was implemented.

:cl:
 * rscadd: Advanced Security Cameras now show cyborg cameras, just like their non-advanced counterpart.

### CODE REVIEW

**Comment by Kammerjunk** [comment]
1. Enable hotkeys
2. Hold down movement key
3. Hit tab
4. Let go of movement key

The client should now be moving in that direction until you hit the movement key again (either with hotkeys or with the arrow keys)

**Comment by Shadowmech88** [comment]
@Unknown-as-Captain 
Have you used the suggested method to test two cyborgs moving at once?
Alternatively, would you like to use the PR test server to test it with multiple people?

**Comment by ghost** [comment]
I won't be available to test this until 2 days from now, if it ends up only updating one borg at a time it's a one line change

File: code/modules/html_interface/map/adv_sec/adv_camera.dm [hunk 1]
```diff
@@ -164,14 +186,18 @@ var/global/datum/interactive_map/camera/adv_camera = new
 		name = C.c_tag
 		var/area/AA = get_area(C)
 		area = format_text(AA.name)
+		if("Robots" in C.network)
```
Comment (PJB3005) [inline code/modules/html_interface/map/adv_sec/adv_camera.dm]: Make this a define like `CAMERA_NETWORK_ROBOTS`.

**Review by PJB3005 — CHANGES_REQUESTED** [review]

**Comment by ghost** [comment]
Tested multiple cyborgs by creating a sushi belt and spawning 10 borgs in it, works alright.

**Review by PJB3005 — APPROVED** [review]
Nice job fixing all the camera nets while you were at it 👍

**Pull request merged**